main.py

```diff
@@ -18,46 +19,22 @@ def main():
             f"Could not connect to server at {host}:{port}. Is the server running?"
         )
 
-    # 1. Spawn a new player
-    print("Requesting a player to be controlled")
-    player_id = client.get_player()
-
-    if player_id is None:
-        return print("Failed to receive a player to control, quitting!")
-
-    # 2. Subscribe to images and rewards for the spawned player
-    print("Subscribing to player")
-    client.subscribe_to_observation(player_id, ObservationKind.IMAGE, cooldown=0.1)
-    client.subscribe_to_observation(player_id, ObservationKind.REWARDS, cooldown=0.1)
+    env = gym.make("gymnasium_env/Tankwars-v0", client)
+    player_id = env.reset()
 
     # 3. Start sending random controls, requesting new images, and displaying them in a loop
     try:
         while True:
-            # Request the latest image
-            client.request_observation(player_id, ObservationKind.IMAGE)
+            player_state = env.get_state(player_id)
```

Comment:
We should get the state from the client

Reply:
Directly from the client, dont use the env (`env.get_state`), when we have direct access to the cline (`client.get_state`)